### PR TITLE
MSPSDS-615: Rename MSPSDS to PSD

### PR DESCRIPTION
**Pull request by broder**
## Description
INB4 bugs and merge conflicts.

Things that are still left to rename are the PaaS space and the repo name.

## Checklist:

- [x] Automated checks are passing locally.
- [x] CHANGELOG updated if change is worth telling users about.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Market Surveillance & Product Safety Digital Service
+# Office for Product Safety & Standards Services
```
Comment (jasiekmiko) [inline README.md]: Do we want to rename the github repo, too?

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ All notable changes to this project will be documented in this file.
 - [ ] Delete mspsds `admin` role (Clients->mspsds-app->roles->admin->delete)
 - [ ] Create the new environment variable services.
 - [ ] Create the `opss-cdn-route` service with the live and deployment URLs.
+- [ ] Rename the `mspsds-app` client to `psd-app`
```
Comment (jasiekmiko) [inline CHANGELOG.md]: also rename the `mspsds_user` role to `psd_user`?

**Comment by jasiekmiko** [comment]
Renaming the realm was a hiccup in a previous release iirc. is this running the risk of similar?

**Comment by kicferk1** [comment]
Does that mean our branches and commits should be PSD-XXXX instead of MSPSDS-XXXX?

File: README.md (hunk 1 above)
Comment (broder) [reply README.md]: We will - I was thinking about doing that a bit later though as it'll probably break Travis and other things like that.

Comment (broder) [reply README.md]: Also, I'm not 100% sure on the repo name right now and didn't really want to hold this up for much longer

File: CHANGELOG.md (hunk 2 above)
Comment (broder) [reply CHANGELOG.md]: Yes, thanks!

**Comment by broder** [comment]
@jasiekmiko, well remembered! The issue was that it re-created the realm because it uses the realm name to check whether or not it already exists so we should have renamed the realm before deploying. I think we'll be okay for this change because we're not touching the realm name

**Comment by broder** [comment]
@kicferk1, not yet. There's probably a separate action to rename the Jira / Confluence space.

File: README.md (hunk 1 above)
Comment (jasiekmiko) [reply README.md]: sounds sensible :)

**Pull request merged**